Assets/Tests/EditModeTests/CommunicationManagerTests.cs

```diff
@@ -21,37 +21,20 @@
 // Jakob Meyer Olsen
 using System.Collections.Generic;
 
-using Maes.Map;
 using Maes.Map.Generators;
 using Maes.Robot;
 
 using NUnit.Framework;
 
-using UnityEngine;
-
 
 namespace Tests.EditModeTests
 {
     [TestFixture]
     public class CommunicationManagerTests
     {
-        private CommunicationManager _communicationManager;
         private RobotConstraints _robotConstraints;
-        private SimulationMap<Tile> _simulationMap;
 
-        private const string TestMap = "" +
-                                    "            ;" +
-                                    " XXXXXXXXXX ;" +
-                                    " X        X ;" +
-                                    " X        X ;" +
-                                    " X        X ;" +
-                                    " X        X ;" +
-                                    " X        X ;" +
-                                    " X        X ;" +
-                                    " X        X ;" +
-                                    " X        X ;" +
-                                    " XXXXXXXXXX ;" +
-                                    "            ";
+        private const string testMap = "sXe;";
```

Comment:
more tests would be nice

Reply:
added a couple more simple test cases, if we add more it will be in another PR